app/lib/fixPhoneNumbers.js

```diff
@@ -0,0 +1,23 @@
+const phoneNumberParser = require('./phoneNumberParser');
```

Comment:
As with the comment about the address above, 'fixing' the phone number feels like it isn't something that fits with what we had talked about wrt the purpose of the ETL. Is it better suited to the `db-updater` layer?
In contrast with the address the phone numbers feels a bit more like formatting rather than fundamental to the ETL...